tangelo/linq/tests/test_simulator.py

```diff
@@ -461,7 +508,8 @@ def __init__(self, n_shots=None, noise_model=None, return_zeros=True):
                 super().__init__(n_shots=n_shots, noise_model=noise_model)
                 self.return_zeros = return_zeros
 
-            def simulate_circuit(self, source_circuit: Circuit, return_statevector=False, initial_statevector=None, save_mid_circuit_meas=False):
+            def simulate_circuit(self, source_circuit: Circuit, return_statevector=False, initial_statevector=None,
+                                 desired_meas_result=None, save_mid_circuit_meas=False):
```

Comment:
Was this returning an error because of inheritance ? I thought python wouldn't bother us with signature match.

Here `initial_statevector=None, desired_meas_result=None, save_mid_circuit_meas=False` are not options supported by the backend and can be removed, unless python complains because of inheritance.

Reply:
Somehow it still needs `initial_statevector=None`, but the mid-circuit stuff can be removed